news/2020-02-20-results-for-monthly-beatmapping-contest-january.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Results for Monthly Beatmapping Contest: January"
+date: 2020-02-20 8:00:00 +0000
+---
+
+The votes are in! Find out the champions of January's Monthly Beatmapping Contest!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+In case you're out of the loop, the **Monthly Beatmapping Contest** (*MBC*) has been revived for a new run in 2020. Read the contest schedule and details on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest). Got everything? Good. Onto the results!
+
+## Winners
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/84).
+
+**Community vote winner:** [**MYLK - Kiradoki Harmony**](https://osu.ppy.sh/beatmapsets/1111940#osu/2323164) by [**melon boy**](https://osu.ppy.sh/users/3053382) (previously known as Fuccho)
```

Comment:
I think the animal names should be included too so the entries can be recognized